scripts/ci-check.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Instead of adding a custom shell script, should we add another setup file in `config/jest`? Could do a `config/jest/feature-flags.js` file:

```js

'use strict';

const FeatureFlags = require.requireActual('../../src/internal/FeatureFlags');

FeatureFlags.componentsX = true;
FeatureFlags.breakingChangesX = true;
```

And the task in yarn would look like:

```json
{
  "scripts": {
    "ci-check": "yarn prettier:diff && yarn lint && yarn test --runInBand && yarn test:experimental --runInBand && yarn test-ssr",
    "test": "jest",
    "test:experimental": "jest --setupFiles='<rootDir>/config/jest/setup.js' --setupFiles='<rootDir>/config/jest/feature-flags.js'",
  }
}
```

Reply:
Makes sense - Updated as such.